### PR TITLE
Resolve #1496, #1497, #1499, #1500, #1501, #1502, #1507, #1505, #1506, #1510

### LINKED ISSUE #1496: [Test] Investigate reduxTracker crashes
This error is infrequent

may be related:
```
(node:10031) MaxListenersExceededWarning: Possible EventEmitter memory leak detected. 11 event listeners added. Use emitter.setMaxListeners() to increase limit
```

### PR DESCRIPTION
**Pull request by timothywangdev**

### CODE REVIEW

File: src/tests/e2e/mocks/drive.js [hunk 1]
```diff
@@ -78,13 +78,16 @@ var data = {
...
 
-var storage = window.localStorage
+var storage
 
-if (!storage.getItem('data')) {
-  // prevent refresh from clearing data
-  storage.setItem('data', JSON.stringify(data))
-}
+function initStorage () {
+  storage = window.localStorage
```
Comment (InVincible2016) [inline src/tests/e2e/mocks/drive.js:84]: #1502 States 'Do not use window in e2e tests in mock drive.js', please update issue description.

File: src/tests/e2e/pages/receipt.page.js [hunk 2]
```diff
@@ -391,11 +391,9 @@ class ReceiptPage {
       .unix(transfer.senderToChainsfer.txTimestamp)
       .format('MMM Do YYYY, HH:mm:ss')
     expect(sendTime).toEqual(`Sent on ${expectSendTime}`)
-    const sendExplorerPage = await getNewPopupPage(browser, async () => {
-      await this.dispatchActions('openSendExplorer')
-    })
-    expect(sendExplorerPage.url()).toEqual(`https://rinkeby.etherscan.io/tx/${transfer.sendTxHash}`)
-    await sendExplorerPage.close()
+
+    const sendLink = (await this.getReceiptFormInfo('sendOn')).sendOnExplorerLink
```
Comment (InVincible2016) [inline src/tests/e2e/pages/receipt.page.js:395]: Why make this change.

File: src/tests/e2e/pages/receipt.page.js [hunk 3]
```diff
@@ -405,13 +403,11 @@ class ReceiptPage {
         .unix(transfer.chainsferToReceiver.txTimestamp)
         .format('MMM Do YYYY, HH:mm:ss')
       expect(receiveTime).toEqual(`Received on ${expectReceiveTime}`)
-      const receiveExplorerPage = await getNewPopupPage(browser, async () => {
-        await this.dispatchActions('openReceiveExplorer')
-      })
-      expect(receiveExplorerPage.url()).toEqual(
+
+      const receiveLink = (await this.getReceiptFormInfo('receiveOn')).receiveOnExplorerLink
```
Comment (InVincible2016) [inline src/tests/e2e/pages/receipt.page.js:407]: Shouldn't we make sure the button actually opens the explorer page?

File: src/tests/e2e/pages/receipt.page.js [hunk 4]
```diff
@@ -422,13 +418,11 @@ class ReceiptPage {
         .unix(transfer.chainsferToSender.txTimestamp)
         .format('MMM Do YYYY, HH:mm:ss')
       expect(cancelTime).toEqual(`Cancelled on ${expectCancelTime}`)
-      const cancelExplorerPage = await getNewPopupPage(browser, async () => {
-        await this.dispatchActions('openCancelExplorer')
-      })
-      expect(cancelExplorerPage.url()).toEqual(
+
+      const cancelLink = (await this.getReceiptFormInfo('cancelOn')).cancelOnExplorerLink
```
Comment (InVincible2016) [inline src/tests/e2e/pages/receipt.page.js:422]: Shouldn't we make sure the button actually opens the explorer page?

Comment (timothywangdev) [reply src/tests/e2e/pages/receipt.page.js:422]: frequent access to etherscan causing use to be blocked and navigation timeout

File: src/tests/e2e/mocks/drive.js (hunk 1 above)
Comment (timothywangdev) [reply src/tests/e2e/mocks/drive.js:84]: updated

**Pull request merged**